stdlib/public/core/Integers.swift.gyb

```diff
@@ -171,13 +1064,35 @@ extension SignedArithmetic {
 }
 
 
-/// Returns the absolute value of `x`.
+/// Returns the absolute value of the given number.
+///
+/// - Parameter x: A signed number.
+/// - Returns: The absolute value of `x`.
 @_transparent
 public func abs<T : SignedArithmetic>(_ x: T) -> T
   where T.Magnitude == T {
   return x.magnitude
 }
 
+/*
+/// Returns the absolute value of the given number.
+///
+/// The absolute value of `x` must be representable in the same type. In
+/// particular, the absolute value of a signed, fixed-width integer type's
+/// minimum cannot be represented.
+///
+///     let x = Int8.min
+///     // x == -128
+///     let y = abs(x)
+///     // Overflow error
+///
+/// - Parameter x: A signed number.
+/// - Returns: The absolute value of `x`.
+public func abs<T : SignedArithmetic>(_ x: T) -> T {
```

Comment:
This doesn't work, since `SignedArithmetic` doesn't require `Comparable`—just a placeholder for documentation of an `abs(_:)` that works for signed integers.